includes/emails/class-give-email-tags.php

```diff
@@ -186,8 +188,31 @@ public function do_tag( $m ) {
  * @param callable $func        Hook to run when email tag is found
  * @param string   $context     Email tag category
  */
-function give_add_email_tag( $tag, $description, $func, $context = '' ) {
-	Give()->email_tags->add( $tag, $description, $func, $context );
+function give_add_email_tag( $args ) {
```

Comment:
@ravinderk How should I update the documentation? How to delete the old one while still keeping the backward-compatibility?

Reply:
@Sidsector9 update doc with the latest param and add `@since` tag and explain in the description what we changed in that specific version